rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -8425,6 +8425,182 @@ private static Stream<Arguments> getEstimatedFeesForNextPegOutEventArgsProvider_
         );
     }
 
+    private static Stream<Arguments> getEstimatedFeesForNextPegOutEventArgsProvider_post_RSKIP305(BridgeConstants bridgeConstants) {
+
+        ActivationConfig.ForBlock tbd800Activations = ActivationConfigsForTest.tbd800().forBlock(0L);
```

Comment:
Let's use `all()` so we ensure this keeps working even after version 8.0.0

Reply:
Updated.